index.html

```diff
@@ -190,11 +190,11 @@ <h3 class="connect">LET’S CONNECT</h3>
 						<li class="sm"><a href="https://www.linkedin.com/in/pratap-kumar-panabaka-755489236/"><img
 									src='images/linkedin.svg' width='20' height='20' alt="LinkedIn Profile"
 									title=''></a></li>
-						<li class="sm"><a href=""><img src='images/v.svg' width='20' height='20' alt='' title=''></a>
+						<li class="sm"><a href=""><img src='images/v.svg' width='20' height='20' alt='Angellist Profile' title=''></a>
 						</li>
 						<li class="sm"><a href="https://twitter.com/PRATAP_TWT"><img src='images/twitter.svg' width='20'
 									height='20' alt="Twitter Profile" title=''></a></li>
-						<li class="sm"><a href=""><img src='images/m.svg' width='20' height='20' alt='' title=''></a>
+						<li class="sm"><a href=""><img src='images/m.svg' width='20' height='20' alt='' title='Medium Profile'></a>
```

Comment:
Social icons Medium and angellist does not have alt text on line 193 - 197. Corrected.